src/modules/project/pages/projectView/views/projectMainBody/sections/Header.tsx

```diff
@@ -169,7 +170,8 @@ export const Header = forwardRef<HTMLDivElement>((_, ref) => {
           </HStack>
           <CreatorSocial />
           {isMobile && (
-            <VStack w="full" paddingTop="5px">
+            <VStack w="full" paddingTop="5px" pb={5}>
+              <MobileMilestoneRecap />
```

Comment:
Is this supposed to be visible only for mobile?

Reply:
Yes, the design is for mobile, otherwise, when on desktop the same information is available elsewhere